conf/modules/call_variants.config

```
@@ -69,11 +69,16 @@ process {
         ]
     }
 
-    withName: '.*BCFTOOLS_STATS' {
+    withName: '.*CALL_VARIANTS:BCFTOOLS_STATS' {
         publishDir = [
             path: { "${params.outdir}/call_variants" },
             mode: params.publish_dir_mode,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
         ]
     }
+
+    withName: '.*CALL_VARIANTS:BCFTOOLS_MERGE' {
+        ext.args = '--output-type z'
+        ext.prefix = { "${meta.id}_merged" }
```

```suggestion
        ext.prefix = { "${meta.id}" }
```
No need to add merged since it should have the case_id and not the sample_id

---

Also this file doesn't seem to get published 🤔

---

Right found it now. It get's published to the bcftools and tabix folder. Could you redirect it to the call_variants folder?